botbot/botbotbot.py

```diff
@@ -286,6 +308,9 @@ def handle_nickreply(self, data):
         super().handle_nickreply(data)
         if self.nickname != previous_nickname:
             self.write_to_file()
+            # update default help and pause texts
+            self.help_text = self.make_help_text()
+            self.generic_pause_text = self.make_generic_pause_text()
```

Comment:
Better than before for sure, but maybe we should consider just generating it in the few places it's needed, i.e.,:
```diff
@@ -117 +117 @@ class BotBotBot(eu.ping_room.PingRoom, eu.chat_room.ChatRoom, eu.nick_room.NickR
-        self.send_chat(self.generic_pause_text, reply_to)
+        self.send_chat(self.make_generic_pause_text(), reply_to)
@@ -182 +182 @@ class BotBotBot(eu.ping_room.PingRoom, eu.chat_room.ChatRoom, eu.nick_room.NickR
-                self.send_chat(self.help_text, this_message)
+                self.send_chat(self.make_help_text(), this_message)
```

Then we can also get rid of the `self.help_text` and `self.generic_pause_text` properties entirely. Not needed for this PR, though.